htdocs/comm/action/class/actioncomm.class.php

```diff
@@ -513,7 +513,7 @@ function fetch($id, $ref='',$ref_ext='')
         $sql.= " a.fk_contact, a.percent as percentage,";
         $sql.= " a.fk_element, a.elementtype,";
         $sql.= " a.priority, a.fulldayevent, a.location, a.punctual, a.transparency,";
-        $sql.= " c.id as type_id, c.code as type_code, c.libelle,";
+        $sql.= " c.id as type_id, c.code as type_code, c.libelle, c.color,";
```

Comment:
Because, here c.color is color dedicated of type of event, can you instead use
"c.color as type_color" ?